controllers/order.js

```diff
@@ -24,7 +24,9 @@ const postOrder = async (req, res)=>{
 }
 const getAllOrder = async (req, res)=>{   
     // refer to graphql part
+    // why I cannot write res.status(200).send(await queryAllOrder()) to pass auto test
```

Comment:
**queryAllOrder()** queries the **database**, you **cannot** write a **unit test** for a method that works with a **database** (unless it's an in-memory database). If you want to test a database you have to write an **integration test**, but we won't have time for that

Reply:
But why "GET menu" can pass unit test? We don't write unit test fro "GET menu", right?

Reply:
actually there are unit tests only for two endpoints (**restaurant** and **order**), see here

https://github.com/konnovdev/food-order-backend/blob/master/test/app.test.js